docs/photoniq/vwrs/vwrs-metrics.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 100
+title: Get VWRs Metrics
+---
+
+Macrometa PhotonIQ Virtual Waiting Rooms (VWRs) has built-in metrics that allow you to retrieve waiting room telemetry per waiting room, per minute. Data is stored for the past 90 days.
+
+Using VWRs metrics, you can:
+
+- Evaluate peak traffic flow through your waiting room and onto your site.
+- Determine how long users spent in the waiting room.
+- Use analytics to help calibrate your waiting room settings.
+
+## Get Metrics
+
+You can retrieve metrics using the following endpoints:
+
+- Get Metrics for a Domain - Retrieve the metrics for a specific domain. Set start and end times to get metrics for a specific time period.
+- Get All Metrics - Retrieve metrics for all domains. Set start and end times to get metrics for a specific time period.
+
+## Available Metrics
+
+The metrics response body includes the following properties:
+
+- **avg_waiting_time**: (integer) The average time users spend in the virtual waiting room before being granted access to the origin service (in milliseconds).
```

Comment:
I would remove the (type) after each description.  This is a very explicit thing that probably is not needed here.